sklearn/cross_validation.py

```diff
@@ -1215,9 +1216,9 @@ def cross_val_predict(estimator, X, y=None, cv=None, n_jobs=1,
         - An object to be used as a cross-validation generator.
         - An iterable yielding train/test splits.
 
-        For integer/None inputs, if ``y`` is binary or multiclass,
-        :class:`StratifiedKFold` used. If the estimator is a classifier
-        or if ``y`` is neither binary nor multiclass, :class:`KFold` is used.
+        For integer/None inputs, if the estimator is a classifier and ``y`` is
+        either binary or multiclass, :class:`StratifiedKFold` used. In all
```

Comment:
nitpick: I think the "y is either binary or multiclass" is redundant right? If the estimator is classifier, then y has to be binary or multiclass. (or at least since https://github.com/scikit-learn/scikit-learn/pull/5783)